### PR TITLE
Agnosticize servo signal wire color in AI tutorial

### PR DESCRIPTION
**Pull request by JessamyT**
Since this came up in Discord

### CODE REVIEW

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Always disconnect devices from power before plugging, unplugging, moving wires,
 {{% /alert %}}
 
 Power off your rover.
-Wire your servo to the Pi by attaching the black wire to ground, red wire to [an available 5V pin](https://pinout.xyz/pinout/5v_power), and yellow wire to [pin 8](https://pinout.xyz/pinout/pin8_gpio14).
+Wire your servo to the Pi by attaching the black wire to ground, red wire to [an available 5V pin](https://pinout.xyz/pinout/5v_power), and signal wire (often yellow) to [pin 8](https://pinout.xyz/pinout/pin8_gpio14).
```
Comment (npentrel) [inline docs/tutorials/projects/integrating-viam-with-openai.md:67]:
```suggestion
Wire your servo to the Pi by attaching the black wire to ground, red wire to [an available 5V pin](https://pinout.xyz/pinout/5v_power), and signal wire (often yellow) to [pin 8](https://pinout.xyz/pinout/pin8_gpio14).
If your servo wires are attached to one another and the order does not match pins on the board, you can use male-female jumper wires to connect them. 
```

Comment (npentrel) [reply docs/tutorials/projects/integrating-viam-with-openai.md:67]: I think adding the jumper wired info might also be nice

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 2]
```diff
@@ -64,7 +64,8 @@ Always disconnect devices from power before plugging, unplugging, moving wires,
...
+If your servo wires are attached to one another and the order does not match pins on the board, you can use male-female jumper wires to connect them.
```
Comment (JessamyT) [inline docs/tutorials/projects/integrating-viam-with-openai.md:68]:
```suggestion
If your servo wires are attached to one another and the order does not match pins on the board, you can use male-female jumper wires to connect them.
```

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 3]
```diff
@@ -64,7 +64,8 @@ Always disconnect devices from power before plugging, unplugging, moving wires,
...
```
Comment (npentrel) [inline docs/tutorials/projects/integrating-viam-with-openai.md:68]:
```suggestion
If your servo wires are attached to one another and the order does not match the pins on the board, you can use male-female jumper wires to connect them.
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**